ContextApp/src/main/java/com/github/migi_1/ContextApp/PositionHolder.java

```diff
@@ -0,0 +1,64 @@
+package com.github.migi_1.ContextApp;
```

Comment:
Do we really want this to be a singleton and not an instance in the MainActivity?